SPECS/time/time.spec

```diff
@@ -38,25 +38,25 @@ Release:    9%{?dist}
 # m4/stdnoreturn.m4:        FSFULLR
 # maint.mk:                 GPLv3+
 # tests/time-posix-quiet.sh:    GPLv3+
-License:    GPLv3+ and GFDL
+License:        GPLv3+ AND GFDL
```

Comment:
The comments above suggest a couple more licenses. Could you double-check, please and either update the `License` or the comments?